admin/src/main/resources/application-logging.yml

```
@@ -0,0 +1,55 @@
+    activate:
+      on-profile: prod-logging
+
+logging:
+  level:
+    root: info
+
+feign:
+  client:
+    config:
+      default:
+        logger-level: basic
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)